Cauldron/Assets/Scripts/RecipeBook.cs

```diff
@@ -135,6 +141,13 @@ public void RecordRecipe(Recipe recipe)
             unlockedRecipes.Add(recipe);
             LockedRecipes.Remove(recipe);
             recipeProvider.SaveRecipes(unlockedRecipes);
+            
+            int eightyPercent = (int)((allMagicalRecipes.Count + allHerbalRecipes.Count) * 0.8);
```

Comment:
Magic number:))) Please make 0.8 a constant at the top of the script and rename this int to something like "isEnoughRecipesUnlocked")) Or better yet - make a variable in the MainSettings)
Also I think that this functionality doesn't really belong here, but since it needs refactoring all the same, it's okey for now)